### ISSUE TITLE
[1.4.1] Failed to verify deployment 

### ISSUE THREAD

**Issue by s7dhansh** [zodern/meteor-up#891]
Mup version (`mup --version`): 1.4.1
App is deployed and is working, but verification fails with following error (in verbose mode):
```
(7) Failed to connect to 172.17.0.2 port 4450: Connection refused   
```                                                   

If I cancel verification, app works.
v1.3.7 works fine.

1.4.1 works for other apps.

One probable difference in case of failures: I am deploying multiple apps on one server.

Please help on debugging this further / fixing this.

**Comment by zodern** [comment]
Please show me your mup config (you can use `mup validate --scrub`) and the full output after `Verifying Deployment`.

**Comment by zodern** [comment]
I just released 1.4.2 which fixed several bugs with `Verifying Deployment`.
